src/main.cpp

```diff
@@ -45,9 +45,12 @@ int main( int argc, char* argv[] ) {
     // Create Camera
     Shader* program = new Shader( "shaders/vertex.glsl", "shaders/fragment.glsl" );
     Shader* programOutline = new Shader( "shaders/vertex.glsl", "shaders/outline.frag" );
+    Shader* programLine = new Shader( "shaders/vertex.glsl", "shaders/linefrag.frag");
 
     // Initial Mesh
     Geometry* geometry = new Geometry( "models/cube/cube.obj", GL_TRIANGLES, program, programOutline );
+    // sketching geometry
+    Geometry* sketch = new Geometry( {}, {}, {}, GL_LINE_STRIP, programLine, programOutline );
```

Comment:
sketch probably doesn't need a stencil shader so just pass in a nullptr (if it doesn't error out)